### ISSUE TITLE
Sana checkpoint trained with SD-3 VAE

### ISSUE THREAD

**Issue by srikarym** [NVlabs/Sana#70]
Hi,
Thank you for open-sourcing your code and trained models. Could you release the Sana text2image model trained with either SD-XL or SD-3 VAE?

**Comment by recoilme** [comment]
Yes, DC-AE is fast, but ruined image details, we also try f64 - same result( We try realistic, anime - https://imgsli.com/MzI0MDg3

May be take a look at auraflow vae?  it's opensourced as i know, comparable to flux/sd3 vae and significantly better DC-AE 

Auraflow vae
![_auraflow](https://github.com/user-attachments/assets/ab1d5f8d-04c5-40b3-b10e-a8cb2b5766e3)

DC-AE
![_auraf64](https://github.com/user-attachments/assets/8f0d3761-a2c8-48c6-8130-89c845a598c6)

Or may be train small model for converting from DC-AE to AuraFlow in latent space directly, what do you think, is this possible?

**Comment by srikarym** [comment]
Makes sense. It's surprising that both Sana obtains better FID scores with this VAE, despite worse reconstruction results.

From the paper:
```although AE-F8C16 exhibits the best reconstruction ability (rFID: F8C16<F16C32<F32C32), we empirically find that the generation results of F32C32 are superior```

I wish they'd release checkpoints trained with other VAEs, allowing users to choose the one that works best for their specific dataset when fine-tuning.

**Comment by srikarym** [comment]
@lawrence-cj Do you plan to release Sana trained with f8c4 / f8c16 VAEs?

**Comment by srikarym** [comment]
@lawrence-cj gentle reminder
